CharacterCreation/01_Races.txt

```diff
@@ -120,87 +149,131 @@ Starting Stat Points: 40
-    Enhanced Speed: +1 movement.
-    Enhanced Strength: +1 STR, metal arms, legs, or both.
-    Neural Implant: +1 Intelligence.
-    Speed Geared Fine Motor Controls: +1 DEX, no augment to movement speed.
-    Skeletal Support Structure: +1 FORT.
-    Implanted Weapon: Pick a pistol from the basic weapons list. This pistol 
-    is implanted in a limb. You may manipulate your limb to use this weapon at
-    the cost of 1 action. While the weapon is out, that limb does not 
-    operate at its normal capacities. Hands would not grasp, legs would not 
-    superbly support you, etc. Subrtact the price of the weapon from your 
-    class starting money. The weapon can be upgraded to another pistol with a 
-    DC 25 Biomechanics check, or a DC 50 Mechanics check. It can be upgraded 
-    to any CQB weapon with a DC 50 Biomechanics or DC 75 Mechanics check.
-    Integrated multitool: Your fingers hide a basic toolkit, including low-
-    speed motorized screwdrivers, soldering iron, wrenches, quantum spanner, 
-    toothpick, pliers, sharp edge, lock pick, etc. If it fits in a leatherman,
-    its plausible that it would be in this toolkit.
-    Internal Reinforcing: Critical Damage threshold +5, +5 base health.
-    External Reinforcing: 5 DR passive, potential for RP abilities.
-
-Cyborgs share all weaknesses with their base race, generally humans. They 
+
+* Occular Augmentation: +1 Perception, Darkvision 20m.
+* Enhanced Speed: +1 movement.
+* Enhanced Strength: +1 STR, metal arms, legs, or both.
+* Neural Implant: +1 Intelligence.
+* Speed Geared Fine Motor Controls: +1 DEX, no augment to movement speed.
+* Skeletal Support Structure: +1 FORT.
+* Implanted Weapon: See Below
+* Integrated multitool: See Below
+* Internal Reinforcing: Critical Damage threshold +5, +5 base health.
+* External Reinforcing: 5 DR passive, potential for RP abilities.
+
+    Once at level 3, and every 5 levels thereafter a Cyborg may purchase an 
+additional augment at the cost of $2000. They must see a cybernetics expert to
+have this augment installed.
+
+== Implanted Weapon ==
+
+    Pick a pistol from the basic weapons list. This pistol is implanted in a 
+limb. You may manipulate your limb to use this weapon at the cost of 1 action. 
+While the weapon is out, that limb does not operate at its normal capacities. 
+Hands would not grasp, legs would not superbly support you, etc. Subrtact the 
+price of the weapon from your class starting money. The weapon can be upgraded 
+to another pistol with a DC 25 Biomechanics check, or a DC 50 Mechanics check. 
+It can be upgraded to any CQB weapon with a DC 50 Biomechanics or DC 75 
+Mechanics check.
+
+== Integrated Multitool ==
+
+    Your fingers hide a basic toolkit, including low-speed motorized 
+screwdrivers, soldering iron, wrenches, quantum spanner, toothpick, pliers, 
+sharp edge, lock pick, etc. If it fits in a leatherman, its plausible that it 
+would be in this toolkit.

```

Comment:
Aren't "Integrated Multitool" and "Implanted Weapon" cybernetic augmentations? This listing makes it look like cyborgs come with them.

Reply:
They are. Please note how the listing for them say to "see below" Where this is the "below". Side note; I really don't like how we reference the Leatherman brand. But that would be scope creep.

Reply:
Ah, I see them in the list now.

I still think it will cause confusion, they way you set them out from all the others with sub-headings. 
Is this too hard on the parser?:
![image](https://user-images.githubusercontent.com/2192156/33746205-9be209de-db70-11e7-870e-9753d148e63f.png)

Yeah, Leatherman probably should be removed. Speaking of scope creep, this doc needs a spell check.

Reply:
There's two reasons not to include the whole segment in the list. The first is that the parser doesn't track indents yet. But we can fix that, it would just take time and a rearchitecture. 
But the second reason is that it won't display well. The typography of having a list with 6 small items and two other items in the list which are much larger is all wrong. It won't scan well. Even in the best display case, the CSS will move things weird places when displayed on devices with different size screens.